my code for login screen

Fajeeer opened the pull request.

Fajeeer committed:
my code for login screen

michaeliwaikato commented on frontend-react/package.json:
```diff
@@ -6,6 +6,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "mobx": "^5.15.6",
+    "mobx-react": "^6.3.0",
+    "modulo": "0.0.3",
```
Not really sure what these are used for or if they are even necessary?

michaeliwaikato commented on frontend-react/src/components/landingpage.js:
```diff
@@ -1,12 +1,58 @@
 import React, { Component } from 'react';
+import Navigation from '../navigation.js';
```
Is there any particular reason this is here?

michaeliwaikato commented on frontend-react/src/components/landingpage.js:
```diff
@@ -1,12 +1,58 @@
 import '../style/landingpage.css';
 
 class LandingPage extends Component {
+    handleClick() {
+        console.log('Click happened');
+    }
```
This code is not being used so it shouldn't be here.

michaeliwaikato commented on frontend-react/src/components/landingpage.js:
```diff
@@ -1,12 +1,58 @@
         return (
-            <div><h1 className="code">Landing Page</h1></div>
+            
+            <div style={{display: "flex"}}>
+                <div style={{width: "50%"}}>
+                    <img src={require('../style/link2.png')}></img>
+                </div>
+                <form style={{width: "50%", padding: "1rem"}} >
+                    <h1 style={{color: "white"}}>User Friendly and nbsp</h1>
+                    <h2 style={{color: "white"}}>Deep Learning</h2>
+
+                    <div style={styles.rectangle}>
+                        <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginLeft: "1rem"}, {marginBottom: "2rem"}} for= {'username'}>Username</label>
+                            <input id={'username'}></input>
+                        </div>
+
+                         <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginRight: "0.1rem"}} for={'password'}>Password</label>
+                            <input id={'password'} type={'password'}></input>
+                        </div>
+
+                        <div style={{margin: "1rem 0 1rem 0"}}>
+                            <label style={{marginRight: "2.5rem"}} for={'port'}>Port</label>
```
Styling should be applied in style/landingpage.css
The source images should also be given a better name and be put in a new folder such as "resources" or something like that.

michaeliwaikato commented on frontend-react/src/components/landingpage.js:
```diff
@@ -1,12 +1,58 @@
     }
 }
-
+const styles = {
+    rectangle: {
+        width: '500px',
+        height: '170px',
+        padding: '50px',
+        marginLeft: '70px',
+        backgroundColor: 'white',
+        borderRadius: 10
+    }
+}
```
Same as above, this should be in style/landingpage.css

Fajeeer committed:
updated login form

Fajeeer committed:
favicon

andrei-msd commented on frontend-react/src/components/landingpage.js:
```diff
@@ -1,12 +1,42 @@
+                    <img className="logo" src={Logo}></img>
+                </div>
+                <form calssName="form">
+                    <h1 className="head1">User Friendly and nbsp</h1>
```
The h1 should be "User Friendly Deep Learning"

andrei-msd replied:
```suggestion
                    <h1 className="head1">User Friendly</h1>
```

Fajeeer committed:
IP address input and styling editing-login page

Fajeeer committed:
changed the login page design to the original

Commit:
Created login page that is React compatible

This commit creates the login page with the respective login
fields. This has been converted from regular HTML and CSS to be
compatible with React.

Fajeeer committed:
Editing favicon

michaeliwaikato committed:
Created login page that is React compatible

This commit creates the login page with the respective login
fields. This has been converted from regular HTML and CSS to be
compatible with React.
